### ISSUE TITLE
allow user to report a translation mistake

### ISSUE THREAD

**Issue by Marc-marc-marc** [openstreetmap/iD#4714]
a user report on irc a translation error but she doesn't know how to report it.
is there a button somewhere for a lamba user to report a translation mistake or that need to be improved ?
maybe this would be a good idea (but it could only concern translation errors, no need to report untranslated strings)

**Comment by hikemaniac** [comment]
iD already has a "Help translate" button. When clicking the ["How to translate" section](https://github.com/openstreetmap/iD/blob/master/CONTRIBUTING.md#translating) the "How to contribute" file opens, which tells everything on how to add and fix translations.

![id translations](https://user-images.githubusercontent.com/31667811/35187519-5d842348-fe25-11e7-9031-bb9aa310ed6c.png)

**Comment by bhousel** [comment]
Yes like @hikemaniac indicated, the translation button was added to push users towards Transifex, but I think we can probably do better.

**Comment by Marc-marc-marc** [comment]
I see two issues : 
the button is too small..... if the size of the screen allows it, could the free space of the status bar contain the text next to the button ?
some users are not able to propose a new translation, but on the other hand they can notice an wrong sentence if a link "report an error" would allow to create a kind of note as there is for the osm data but concerning the iD software or its translation, this should ideally also be possible for those who do not have/want a github or transifex account.

**Comment by bhousel** [comment]
I actually want to get rid of that translation button.  Maybe a better thing would be to have the bug button pop up a small screen to ask the user what they want to report an issue with "the editor, the map, the imagery, the translation, something else?" and direct them to report their issue in the right place.

**Comment by Marc-marc-marc** [comment]
it look like a good idea.
